doc/nrf/ug_nrf9160.rst

```diff
@@ -272,29 +288,29 @@ If you need to restore the original firmware at some point, download the nRF9160
 To program the HEX file, use nrfjprog (which is part of the `nRF Command Line Tools`_).
 
 If you want to route some pins differently from what is done in the preprogrammed firmware, program the :ref:`zephyr:hello_world` sample instead of the preprogrammed firmware.
-Build the sample (located under ``ncs/zephyr/samples/hello_world``) for the nrf9160dk_nrf52840 board.
-To change the routing options, enable or disable the corresponding devicetree nodes for that board as needed.
+Build the sample (located under ``ncs/zephyr/samples/hello_world``) for the nrf9160dk_nrf52840 development kit.
+To change the routing options, enable or disable the corresponding devicetree nodes for that development kit as needed.
```

Comment:
nrf9160dk_nrf52840 is not a development kit and such term should not be used in this context. According to https://github.com/nrfconnect/sdk-nrf/pull/3708, it is a board. Perhaps "target" could be also used here, as this part refers to building.

Reply:
Correct, this should be board.

Reply:
fixed